### ISSUE TITLE
Count Start Time causing jQuery error if enabled as time_input control

### ISSUE THREAD

**Issue by andrewvanbreda** [BiologicalRecordsCentre/PoMS#17]
If the Count Start Time attribute is set to be a "time_input" in the Warehouse attribute settings this causes an error on the input_form saying (even if not using Firefox)

jQuery.Deferred exception: undefined is not an object (evaluating '$.browser.mozilla')

**Comment by andrewvanbreda** [comment]
Hi @BirenRathod 

Very low prority, but do you know if iRecord has a jQuery plugin called timeentry installed?

I think the "Start time (BST) for your count" control on the following page maybe using it, any thoughts on getting this installed on POMS? As I say, low priority as the field can accept entry without the fancy spinner.

https://www.brc.ac.uk/irecord/poms-fit-count

**Comment by andrewvanbreda** [comment]
Closing this as John says this
"I think I've setup time inputs before where I just used a text_input with the regex validation on the attribute so it is forced to hh:mm format.
It's quite likely that I'll re-code the actual time_input to a html5 input with type="time" at some point, rather than use a library. That matches what we've done with dates. However for Safari we'll need a Polyfill or alternative at this point."

For now I switched on the Warehouse time validation on the Warehouse so a valid time must be entered on the text input.
